src/components/DateTimePicker/pickerModal.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Calendar from '../Calendar';
+import TimeSelect from '../TimePicker/timeSelect';
+import extractDate from './helpers/extractDate';
+import extractTime from './helpers/extractTime';
+import formatDate from '../DatePicker/helpers/formatDate';
+import Styled from './styledComponents';
+
+function DateTimePickerModal(props) {
+    const {
+        isOpen,
+        title,
+        value,
+        minDate,
+        maxDate,
+        formatStyle,
+        okLabel,
+        cancelLabel,
+        onRequestClose,
+        onChange,
+    } = props;
+
+    const [date, setDate] = useState(value);
+    const modalTitle = title || formatDate(date, formatStyle);
```

Comment:
right now there is not title passed from parent but
```formatDate(date, formatStyle)```
is already called in parent component so pass down the formattedDatetime thus there is not duplicate computation